docs/reference/query-dsl/query-string-query.asciidoc

```diff
@@ -4,163 +4,251 @@
 <titleabbrev>Query string</titleabbrev>
 ++++
 
-A query that uses a query parser in order to parse its content. Here is
-an example:
+Returns documents based on a provided query string, using a parser with a strict
+syntax.
+
+This query uses a <<query-string-syntax,syntax>> to parse and split the provided
+query string into terms based on operators, such as `AND` or `NOT`. The query
+then <<analysis,analyzes>> each term independently before returning matching
+documents.
+
+You can use the `query_string` query to create a complex search that includes
+wildcard characters, searches across multiple fields, and more. While versatile,
+the query is strict and returns an error if the query string includes any
+invalid syntax.
+
+[WARNING]
+====
+Because it returns an error for any invalid syntax, we don't recommend using
+the `query_string` query for search boxes.
+
+If you don't need to support a query syntax, consider using the
+<<query-dsl-match-query, `match`>> query. If you need the features of a query
+syntax, use the <<query-dsl-simple-query-string-query,`simple_query_string`>>
+query, which is less strict.
+====
+ 
+
+[[query-string-query-ex-request]]
+==== Example request
+
+When running the following search, the `query_string` query splits `(new york
+city) OR (big apple)` into two terms: `new york city` and `big apple`. The
```

Comment:
> into two terms

`term` has another special meaning in ES/Lucene; a term is an output from an analyzer.  Let call these "into two parts" or something like this.